### ISSUE TITLE
[BUG] disabled text area with calculated value doesnt show content on multiple lines

### ISSUE THREAD

**Issue by gismomtl** [formio/formio.js#2471]
## Environment
Please provide as many details as you can:

- Hosting type
  - [] Form.io
  - [X] Local deployment
    - Version: 1.70
- Formio.js version:4.9.0.rc2
- Angular-Formio : 4.5.8

inside a TextArea component, the calculated value is a concatenation of multiple field value separated by an end of line (\n)
with a not-disabled component, text is displayed correctly with return carriage.
However when component is disabled, it's not working.. everything is put in the same line

cf https://jsfiddle.net/4vahzLxc/

**Comment by travist** [comment]
This is most likely expected. If you want it to show in multiple lines, change your textarea to use a wysiwyg editor.

**Comment by gismomtl** [comment]
With wysiwyg option enabled, it doesn't solve the issue. ( https://jsfiddle.net/4vahzLxc/ has been updated with  wysiwyg option enabled

**Comment by llaurentiu** [comment]
Hi @travist . 
The defect is still reproducible (I checked 4.12.x. and 5.0.0-m.10).

It is not reproducible when we view a submission in the Formio builder, because the following css rule applies:
`div[read-only="true"] .formio-component-textarea div[ref="input"] { white-space: pre-wrap; }`. However, this will not apply when the text area is rendered in a modal.

When rendered in our app, the wrapping div does not exist.

I created a jss fiddle that inserts the required css and fixes the issue for both modal and normal rendering: https://jsfiddle.net/darkdante/zxo8ykma/25/ 

Another solution would be to render `<br/>` for endlines in read only mode.

Can this be included in one of the upcoming releases ?
